services/bitcoin.js

```diff
@@ -33,20 +33,19 @@ const Service = require('../types/service');
 const State = require('../types/state');
 const Chain = require('../types/chain');
 const Wallet = require('../types/wallet');
-const Consensus = require('../types/consensus');
+// const Consensus = require('../types/consensus');
 
 // Special Types (internal to Bitcoin)
 const BitcoinBlock = require('../types/bitcoin/block');
 const BitcoinTransaction = require('../types/bitcoin/transaction');
 
 // Convenience Labels
-const Amount = bcoin.Amount;
-const Coin = bcoin.Coin;
-const FullNode = bcoin.FullNode;
-const KeyRing = bcoin.KeyRing;
-const MTX = bcoin.MTX;
-const NetAddress = bcoin.net.NetAddress;
-const Script = bcoin.Script;
+// const Amount = bcoin.Amount;
+// const Coin = bcoin.Coin;
+// const FullNode = bcoin.FullNode;
+// const MTX = bcoin.MTX;
+// const NetAddress = bcoin.net.NetAddress;
+// const Script = bcoin.Script;
```

Comment:
dead code removal?

Reply:
Addressed in 11d4429d6a73eb4c50bea7d01999fc2bd3e4d8b1 (included with #93)